GOVERNANCE.md

```diff
@@ -182,6 +182,35 @@ The project Steward is currently: **@FredKSchott**
 - New stewards will be added based on a unanimous vote by the existing Steward(s).
 - In the event that someone is unreachable then the decision will be deferred.
 
+
+## Other Roles
+### Staff
+
+Staff is a special designation for employees of [The Astro Technology Company](https://astro.build/company) that lives outside of our Governance model. The staff role was designed to help those of us working full-time on Astro to work productively without "skipping the line" and circumventing our governance model entirely.
+
+The staff role was designed to offer instant **visibility** and **trust**, but not instant **authority.**
+
+#### Privileges
+
+All privileges of the [Core Maintainer role](#level-3---core-mainainer), except...
+
+- Instead of gaining Discord contributor roles (`@contributor`, `@maintainer`, `@core`) you will receive a special `@staff` role in Discord and GitHub that grants equivalent visibility and permissions as `@core`.
+- No voting abilities for the first 3 months of staff membership. Then, the role grants equivalent voting permissions as `@core` (see [Voting](#voting) below).
+- No longer eligible for additional contributor levels. You are essentially frozen at your current contributor level until you leave staff.
```

Comment:
This was the only thing that jumped out at me, that I want to make sure I understand and that it's what is intended.  :)

It would neither surprise nor bother me if Astro staff "earned" their way into core while staff. This line seems to imply that that is not an option. Both Jon and Tony have brought up this point re: their joining staff, wanting to "earn" core and not just immediately receive the governance status because of their employment. But, this clause seems to prevent that from happen, with their status "frozen" as long as they're employed.

So, just checking on this!

Reply:
@tony-sull that's in line with what I was envisioning here! Let me update this line to look more like what you've written to make the distinction between "community" and "staff" more clear.

@sarah11918 yup, that's intentional. What I'm trying to prevent is a #core channel that is filled with staff and ex-staff members. Because staff is "gifted" the same rights as core, this distinction should have no impact on how staff members can function in our community.

The section on `#### Leaving Staff` is meant to address this by saying that anyone leaving staff is still able to jump up to `@core` through the normal nomination channels if they would like.